src/Mysql/Select.php

```diff
@@ -91,6 +92,19 @@ public function addValue($tableId, $column)
         return self::insert($this->columns, $name);
     }
 
+    public function addCount()
+    {
+        $countExpression = 'COUNT(TRUE)';
+        return self::insert($this->columns, $countExpression);
+    }
+
+    public function addAggregator($aggregator, Column $column)
+    {
+        $columnMysql = $column->getMysql();
+        $aggregatorExpression = "{$aggregator}({$columnMysql})";
```

Comment:
This allows people to inject their own MySQL into our queries.

We could prevent this by adding to the Mysql classes: We already have a "Column" class. We could add a class for each aggregator. Each class would produce its own MySQL, in the same way that the "Column" does.